core/cap-0021.md

```diff
@@ -525,7 +525,14 @@ They then exchange C_i and D_i (which unlike the update case, can be
 exchanged in a single phase of communication because D_i is not yet
 executable while E's sequence number is below the new s).  Finally,
 they create a top-up transaction that atomically adjusts E's balance
-and uses `BUMP_SEQUENCE` to increase E's sequence number to s.
+and uses `BUMP_SEQUENCE` to increase E's sequence number to s. The
+top-up transaction has a source account set to any account other than
+E.
+
+If a top-up transaction by R fails, the state of E is unchanged. The
+participants can choose to close the channel with the previous D_i and
+C_i. If they wish to keep the channel open they discard the change to
+s, increment i, and exchange C_i and D_i containing the agreed state.
```

Comment:
How about "increment i again"?

Reply:
Done in 042d901aa6124bcd5e1a2e3c2bd85e16c9f286e7.